vm/bin/root-bootstrap.sh

```diff
@@ -118,6 +118,12 @@ cd /home/p4
 bash /vagrant/bin/user-bootstrap.sh
 EOF
 
+# make sure all files in /home/p4 are owned by user p4
+chown -R p4:p4 /home/p4/
+
 # Change Vagrant password. Otherwise if deployed in the wild it can be a vulnerability
 # Alternative would be to make vagrant and root users only sshable with a key
-echo "vagrant:gv82NEudNnp$w87[" | sudo chpasswd
\ No newline at end of file
+echo "vagrant:gv82NEudNnp$w87[" | sudo chpasswd
+
+# Do this last!
+reboot
```

Comment:
This seems like a reasonable time to reboot, but note that if changes like these are made, the README instructions become out of date, because they mention using ssh into port 2223, which is now wrong if these changes are made.